kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/StreamsApp.java

```diff
@@ -0,0 +1,70 @@
+package org.hypertrace.core.kafkastreams.framework;
+
+import com.typesafe.config.Config;
+import java.time.Duration;
+import java.util.Properties;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.Topology;
+import org.hypertrace.core.kafkastreams.framework.listeners.LoggingStateListener;
+import org.hypertrace.core.kafkastreams.framework.listeners.LoggingStateRestoreListener;
+import org.hypertrace.core.kafkastreams.framework.util.ExceptionUtils;
+import org.hypertrace.core.serviceframework.background.PlatformBackgroundJob;
+import org.hypertrace.core.serviceframework.config.ConfigUtils;
+import org.slf4j.Logger;
+
+/**
+ * Abstract base class that all Kafka Streams based applications need to extend
+ */
+public abstract class StreamsApp implements PlatformBackgroundJob {
```

Comment:
Can we rename a class name as `KafkaStreamsApp` from `StreamsApp`?

Reply:
done